doc-content/shared-kie-docs/src/main/asciidoc/Install/testing-clustering-environment-proc.adoc

```diff
@@ -0,0 +1,122 @@
+[id='testing-clustering-environment-proc_{context}']
+= Testing that your infrastructure meets the minimum hardware and performance requirements for a viable high availability (HA) on-premises environment.
+
+When creating a production-ready high availability (HA) on-premises infrastructure for {CENTRAL}, you must ensure it meets the minimum hardware and performance requirements for a viable HA environment, and to mitigate performance related issues. This section describes the environment elements, an example environment, and how to test the performance.
+
+There are four main components; {CENTRAL}, the message system (AMQ), the indexing server ({DATAGRID}), and a shared file system (NFS/GlusterFS/Ceph).
+
+.Prerequisites
+
+* A network environment of at least 3 nodes is configured with the following layout:
++
+Node 1: {CENTRAL}
++
+Node 2: {CENTRAL}
++
+Node 3: AMQ, {DATAGRID}, and NFS
+
+.Procedure
+
+. To test the network speed, do the following tasks:
```

Comment:
For all of these steps, instead of "do the following tasks", I'd suggest either "perform the following actions:" or just "follow these steps:". Also, if it were me I'd make them direct actions rather than infinitives that feel like options. I also typically favor "verify" instead of "check", less colloquial. All subjective, strictly my opinion.

"Test the network speed by following these steps:"
"Verify the NFS information by following these steps:"
"Verify that the disk is an SSD by following these steps:".

Reply:
Done